easybuild/easyblocks/m/mcr.py

```diff
@@ -68,12 +68,16 @@ def configure_step(self):
         """Configure MCR installation: create license file."""
 
         configfile = os.path.join(self.builddir, self.configfilename)
-        if LooseVersion(self.version) < LooseVersion('2015a'):
+        if LooseVersion(self.version) < LooseVersion('R2015a'):
             shutil.copyfile(os.path.join(self.cfg['start_dir'], 'installer_input.txt'), configfile)
             config = read_file(configfile)
-            config = re.sub(r"^# destinationFolder=.*", "destinationFolder=%s" % self.installdir, config, re.M)
-            config = re.sub(r"^# agreeToLicense=.*", "agreeToLicense=Yes", config, re.M)
-            config = re.sub(r"^# mode=.*", "mode=silent", config, re.M)
+            regdest = re.compile(r"^# destinationFolder=.*", re.M)
+            regagree = re.compile(r"^# agreeToLicense=.*", re.M)
+            regmode = re.compile(r"^# mode=.*", re.M)
+
+            config = regdest.sub("destinationFolder=%s" % self.installdir, config)
+            config = regagree.sub("agreeToLicense=Yes", config)
+            config = regmode.sub("mode=silent", config)
```

Comment:
Hah, OK, I wasn't aware of that...

We should include a comment to clarify that, can you add something like this @mboisson?

```python
# compile regex first since re.sub doesn't accept re.M flag for multiline regex in Python 2.6
```